auditlog_security/models/ir_rule.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2021 Therp B.V.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import exceptions, models, fields, api, _
+
+
+class IrRule(models.Model):
+    _inherit = 'ir.rule'
+
+    auditlog_id = fields.Many2one(
+        'auditlog.rule',
+        required=False,
+        help="Auditlog Rule that generated this ir.rule")
+
+    @api.multi
+    def write(self, vals):
+        #  for this in self:
+        # if this.auditlog_id and not self.context('auditlog_write'):
+        #     raise exceptions.ValidationError(
+        #             _("""This rule was automatically generated from the auditlog 
+        #             interface, please use that to edit/delete it """))
```

Comment:
I say we allow editing, but not deleting. Editing can come in handy in case of, well, Idunno. Shenanigans.

Reply:
Okay. But we will forbid changing the autolog_id field.  
that is the connection with our rule. it should not be manually changed/lost.

My concern was that there is a difference between the autolog.rule and the ir.rule , that may confuse the user.
but editing is feasable it we accept that